src/commands/deal/create.ts

```diff
@@ -66,11 +66,11 @@ export default class Create extends BaseCommand<typeof Create> {
 
     const dealAddress = await dealCreate({
       appCID: flags["app-cid"],
-      collateralPerWorker: flags["collateral-per-worker"],
+      collateralPerWorker: Number(flags["collateral-per-worker"]),
       minWorkers: flags["min-workers"],
       targetWorkers: flags["target-workers"],
       maxWorkersPerProvider: flags["max-workers-per-provider"],
-      pricePerWorkerEpoch: flags["price-per-worker-epoch"],
+      pricePerWorkerEpoch: Number(flags["price-per-worker-epoch"]),
```

Comment:
Do you validate it somehow? What if string contains float value for example?

Reply:
this command is hidden from users and only for development purposes, so I don't validate anything here